minato_namikaze/discordbot.py

```diff
@@ -187,4 +191,4 @@ async def on_ready(self):
             )
 
 if __name__ == "__main__":
-    MinatoNamikazeBot.run()
+    MinatoNamikazeBot().run()
```

Comment:
hey here no need to do this instead just do `MinatoNamikazeBot.run()` because if you do `MinatoNamikazeBot().run()` you are first calling the init function of that class

Reply:
well, when i run it as 
```py
MinatoNamikazeBot.run()
``` 
it says 
```
TypeError: run() missing 1 required positional argument: 'self'
```

and when i run it as 
```py
MinatoNamikazeBot().run()
``` 
it works fine there. so.. u still want me to change it?

Reply:
okay then noo need to change it, my mad :(

Reply:
haha no worries